pkg/local/utils.go

```diff
@@ -20,10 +20,10 @@ const (
 	STORAGE_DIR_NAME                   = "storage"
 	ELASTIC_STORAGE_DIR_NAME           = "storage/keycloak-postgresql"
 	KECKLOCK_POSTGRES_STORAGE_DIR_NAME = "storage/elasticsearch"
-
-	MANIFEST_DIR_NAME               = "manifests"
-	INSTALLATION_PARAMS_FILE_NAME   = "params.yaml"
-	INSTALLATION_MANIFEST_FILE_NAME = "manifest.yaml"
+	HOSTNAME_FILE                      = "hostname"
```

Comment:
The Domain is already saved on installation params, consider adding datadog customer installation paramas instead of creating a new file

Reply:
Good idea. But is it always given? The domain, I mean... I was under the impression it's only for the HTTPS. If so, and every standalone installation also has a domain name associated with it then I agree it's definitely a more useful tag than hostname.

Reply:
It is not just for `https`, so you can add `IdentifierName` param and flag and use the domain name it it existed as default.

Reply:
The `installation-params` is similar to `installation-flag` bug It is still different, the flag is more for the user interface and installation-params are the final params that we used after mapping the flag and if needed computing or asking the user for more details. so any installation params state are saved as installation-params